google-cloud-datastore/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+### 1.5.3 / 2019-06-11
+
+* Enable grpc.service_config_disable_resolution
+* Add VERSION constant
```

Comment:
Datastore already had a VERSION constant, this change uses the constant in the GAPIC client.

Reply:
Fixed.